manifests/sentinel.pp

```diff
@@ -190,6 +195,7 @@
   $service_group          = $::redis::params::service_group,
   $service_name           = $::redis::params::sentinel_service_name,
   $service_ensure         = $::redis::params::service_ensure,
+  $service_enable         = $::redis::params::service_enable,
```

Comment:
Please add a datatype for the new parameter and a unit test.

Reply:
Datatype added and email address updated, good catch.